Sim/SimG4Components/src/SimG4Svc.h

```diff
@@ -77,8 +84,14 @@ class SimG4Svc : public extends1<Service, ISimG4Svc> {
   Gaudi::Property<std::vector<std::string>> m_regionToolNames{
       this, "regions", {}, "Names of the tools that create regions and fast simulation models"};
 
+  Gaudi::Property<bool> m_interactiveMode{this, "InteractiveMode", false, "Enter the interactive mode"};
+
   /// Run Manager
   sim::RunManager m_runManager;
+
+  std::unique_ptr<G4VisManager> visManager = std::make_unique<G4VisExecutive>();
```

Comment:
One last thing: Could you rename the `visManager` to `m_visManager`, as in the style guide, and only initialize the pointer in `initialize()`? So that would be

`std::unique_ptr<G4VisManager> m_visManager{nullptr};`

and then

`m_visManager = std::make_unique<G4VisExecutive>();`

in the `.cpp`, and analogous for `session`.  Thanks!